pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = "nipanel"
-version = "0.1.0-dev2"
```

Comment:
You don't have to bump minor for this component because it still has major version 0.

https://semver.org/#spec-item-4

> Major version zero (0.y.z) is for initial development. Anything MAY change at any time. The public API SHOULD NOT be considered stable.